.build/README.md

```
@@ -0,0 +1,39 @@
+5. **Integration Tests**: DSC resource integration test files under `tests/Integration/Resources/`
+
+## Usage
+
+### Azure Pipelines
+```yaml
+- powershell: |
+    .build/Test-ShouldRunDscResourceIntegrationTests.ps1
+  displayName: 'Determine if DSC resource tests should run'
+```
+
+### Command Line
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)